tests/happy-path.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { test, expect, _electron as electron } from '@playwright/test';
+import { mkdtemp, rm } from 'node:fs/promises';
+import { tmpdir } from 'node:os';
+import { sep, join } from 'node:path';
+
+let installDir: string;
+
+
+test.beforeAll(async () => {
+  try {
+    installDir = await mkdtemp(`${tmpdir()}${sep}drupal-temp-`);
+    console.log(`Temporary directory created at: ${installDir}`);
+  } catch (err) {
+    console.error('Error creating temporary directory:', err);
+  }
+
+  console.log('Before tests');
+});
+
+test.afterAll(async () => {
+  try {
+    rm(installDir, { recursive:true } );
+    console.log(`Successfully removed: ${installDir}`);
+  } catch (err) {
+    console.error('Error removing temporary directory:', err);
+  }
+
+  console.log('After tests');
+});
+
+test('happy path test', async () => {
+  test.setTimeout(30_000);
+  const launchOptions = {
+    args: [
+      '.',
+      `--root=${join(installDir, 'drupal')}`,
```

Comment:
We could probably get rid of this if, when we call `mkdtemp()`, we do something like:

```
await mkdtemp(path.join(tmpdir(), 'drupal-launcher-tests') + sep);
```

That way, I would think that each test will gets own randomly-named directory, all in a single subdirectory for cleanliness. What do you think?

Reply:
On second thought, I think we might have an even better way given to us for free by Playwright: https://github.com/drupal/cms-launcher/pull/114#issuecomment-3169662145